docs/source/filters.rst

```diff
@@ -5,21 +5,19 @@
 Filters
 =======
 
-Filters in Ansible are from Jinja2, and are used for transforming data inside
-a template expression. Jinja2 ships with many filters as does Ansible and also
-allows users to add their own custom filters.
+Filters in Ansible are from Jinja2, and are used to transform data inside
+a template expression. The templates operate on the Ansible controller, and not
+on the target host. Therefore, filters execute on the controller as they augment
+the data locally.
 
-It should be noted that templates operate on the Ansible controller, not on the
-target host, so filters execute on the controller as they augment the data
-locally.
```

Comment:
i assume the "so" here means therefore. If not, we have to rephrase the sentence again